manifests/prometheus/alerts.d/concourse-check-certificates-errors.yml

```
@@ -0,0 +1,18 @@
+# Source: concourse
+---
+
+- type: replace
+  path: /instance_groups/name=prometheus2/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: ConcourseCheckCertificatesErrors
+    rules:
+      - record: "concourse_check_certificates_errors"
+        expr: sum(concourse_builds_finished{exported_job="check-certificates",pipeline="create-cloudfoundry",status="errored"} or (absent(concourse_builds_finished{exported_job="check-certificates",pipeline="create-cloudfoundry",status="errored"})-1))
```

Is the label `exported_job`? I /think/ it's just `job`, as per https://github.com/alphagov/paas-atc/blob/gds_master/metric/emitter/prometheus.go#L120

```suggestion
        expr: sum(concourse_builds_finished{job="check-certificates",pipeline="create-cloudfoundry",status="errored"} or (absent(concourse_builds_finished{job="check-certificates",pipeline="create-cloudfoundry",status="errored"})-1))
```

---

Your comment assumes that I haven't tested any of these alerts and I just came up with a different label.

But answering the question you should've asked: why is the label different? It seems the `job` label is preserved to contain the Bosh job name and the relabeling config renames the label to `exported_job`.

---

See https://github.com/prometheus/prometheus/blob/e3bdc463fa7f8d8b5dffe02f35a03012d4213fb5/docs/configuration/configuration.md#scrape_config - `honor_labels`

---

Hey @bandesz. I did check through this PR for an explicit relabelling operation as per https://github.com/prometheus/prometheus/blob/master/docs/configuration/configuration.md#relabel_config before commenting, and (obviously!) didn't find one. Thanks for the pointer to that other Prometheus config setting 👍